src/content/en/2021/ecommerce.md

```diff
@@ -2,7 +2,7 @@
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Ecommerce
 description: TODO
-authors: []
+authors: [bobbyshaw]
 reviewers: []
 analysts: []
```

Comment:
@bobbyshaw can you suggest edits to the Authors, Reviewers, and Analysts properties so that they contain everyone who contributed to the chapter, roughly ordered by those with the most significant contributions first? This is what readers will see at the top of the chapter so we want to be sure to give appropriate credit to everyone who helped.

Reply:
Hi Rick. I have these completed in my local environment as part of the markdown drafting process. I shall submit separately if that's ok.

Reply:
Sounds good. Closing this.

Reply:
Actually, since we're doing the early release tomorrow, it would be great if the Contributors page was updated in time to include all of the Ecommerce chapter contributors. Would it be possible to suggest the contributor names? I'll make sure their metadata is updated in 2021.json.